# Target forward momentum in millis s^-1
int16 targetSpeed

bool isRadians
float64 targetClockwiseRadiansRotation
float64 targetClockwiseDegreesRotation